autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -49,9 +49,13 @@ def _fit(
             # with nan values.
             # Columns that are completely made of NaN values are provided to the pipeline
             # so that later stages decide how to handle them
+
+            # Clear whatever null column markers we had previously
+            self.null_columns.clear()
             if np.any(pd.isnull(X)):
```

Comment:
Do we need this condition?
It seems to me that the time complexity for both the inner loop and this condition check are same.

Reply:
yes I think we can get rid of this line